mkosi/__init__.py

```diff
@@ -568,8 +571,23 @@ def script_maybe_chroot_sandbox(
 ) -> Iterator[list[PathString]]:
     options = ["--dir", "/work/src", "--chdir", "/work/src", *options]
 
+    suppress_chown = parse_boolean(context.config.environment.get("MKOSI_CHROOT_SUPPRESS_CHOWN", "0"))
+    if suppress_chown:
+        options += ["--suppress-chown"]
+
     helpers = {
-        "mkosi-chroot": chroot_script_cmd(tools=bool(context.config.tools_tree), network=network, work=True),
+        "mkosi-chroot": [
+            choose_interpreter(bool(context.config.tools_tree)), "-SI", "/sandbox.py",
+            "--bind", "/buildroot", "/",
+            "--bind", "/var/tmp", "/var/tmp",
+            *apivfs_options(root=Path("/")),
+            *chroot_options(),
+            "--bind", "/work", "/work",
+            "--chdir", "/work/src",
```

Comment:
You can pass `options` here as well and get rid of passing `--chdir` and `--suppress-chown` manually here

Reply:
Split out options from the caller and the ones here, which then allows the reuse.